crates/nu-command/src/formats/to/nuon.rs

```diff
@@ -181,6 +186,12 @@ fn to_nuon(call: &Call, input: PipelineData) -> Result<String, ShellError> {
     value_to_string(&v, call.head)
 }
 
+fn needs_quotes(string: &str) -> bool {
+    string.contains(' ') 
+        || string.contains(',') 
+        || string.contains(':')
```

Comment:
Thinking about it, the parentheses, brackets, and braces are probably also on the list (I don't know if we actively deny certain escaped identifiers on the nushell side.). For good measure I also threw the semicolon into it.
```suggestion
        || string.contains(':')
        || string.contains(';')
        || string.contains('(')
        || string.contains(')')
        || string.contains('[')
        || string.contains(']')
        || string.contains('{')
        || string.contains('}')
```